components/search/FacetDropdown.vue

```diff
@@ -0,0 +1,254 @@
+<template>
+  <b-dropdown
+    ref="dropdown"
+    v-click-outside="applySelection"
+    :variant="hasSelection"
+    class="mr-2"
+    :data-type="facetType"
+    data-qa="search facet"
+  >
+    <template v-slot:button-content>
+      <span :data-qa="`${facetName} dropdown button`">
+        {{ facetName }}
+      </span>
+    </template>
+
+    <b-dropdown-form class="options-container">
```

Comment:
Let's change the logic within this element to:
```html
      <template
        v-if="isRadio"
      >
        <b-form-radio
          v-for="(option, index) in sortOptions"
          :key="index"
          ...
        >
          ...
        </b-form-radio>
      </template>
      <template
        v-else
      >
        <b-form-checkbox
          v-for="(option, index) in sortOptions"
          :key="index"
          ...
        >
          ...
        </b-form-checkbox>
      </template>
```
The advantages of this are:
1. The superfluous `<div>` element is avoided
2. The if/else condition is only tested once for each use of the component instead of once for each option